### PR TITLE
Serverless eBook updates

### PR DESCRIPTION
**Pull request by JeremyLikness**
## Summary

Updated eBook to reflect current Azure Functions v3 and update samples.

### CODE REVIEW

**Comment by JeremyLikness** [comment]
These changes are ready for review @nishanil. I believe @cecilphillip will address the durable chapters in a separate PR.

**Comment by JeremyLikness** [comment]
Note to @nishanil and @cecilphillip new cover has been updated ([view it here](https://review.docs.microsoft.com/en-us/dotnet/architecture/serverless/?branch=pr-en-us-17926))

**Review by nishanil — COMMENTED** [review]
@JeremyLikness Just a minor change requested below. Everything else looks good to me. Can we merge this now or do you want Cecil's changes to come in before we go live as Version 3.0?

File: docs/architecture/serverless/index.md [hunk 1]
```diff
@@ -3,12 +3,12 @@ title: "Serverless apps: Architecture, patterns, and Azure implementation"
...
 author: JEREMYLIKNESS
 ms.author: jeliknes
-ms.date: 06/26/2018
+ms.date: 04/22/2020
 ---
 
 # Serverless apps: Architecture, patterns, and Azure implementation
 
-![Screenshot that shows the Serverless Apps e-book cover.](./media/index/serverless-apps-cover.jpg)
+![Screenshot that shows the Serverless Apps e-book cover.](./media/index/serverless-apps-cover-v3.png)
```
[the remark this answers is not in the document]
Comment (JeremyLikness) [reply docs/architecture/serverless/index.md:11]: Done!

**Comment by JeremyLikness** [comment]
@nishanil updates applied, and I'm fine with merging now

**Review by nishanil — APPROVED** [review]

**Pull request merged**